packages/sql-openapi/test/simple.test.js

```diff
@@ -311,13 +311,6 @@ test('list', async ({ pass, teardown, same, equal }) => {
     }).slice(2), `${url} response`)
   }
 
-  {
```

Comment:
I have added after this tests (temporarly removed) in other position in a next commit.